.github/workflows/create-container.yaml

```diff
@@ -0,0 +1,72 @@
+name: Publish GAP Docker image
+
+on:
+  workflow_call:
+    inputs:
+      GAP_VERSION:
```

Comment:
I think all of them should be a `choice` with closed catalog of options.

Reply:
I don't think this is helpful for inputs of a `workflow_call`, since the values are given by the matrix in main.yaml, and this would then require maintaining the same list in two different files?

Reply:
Actually, I agree. In fact, we will have two lists anyway, since we want to have `workflow_dispatch` too.